sdk/boms/azure-spring-boot-bom/pom.xml

```diff
@@ -38,6 +38,10 @@
     <packageOutputDirectory>${project.build.directory}</packageOutputDirectory>
     <legal><![CDATA[[INFO] Any downloads listed may be third party software.  Microsoft grants you no rights for third party software.]]></legal>
     <azure.spring.boot.version>${project.version}</azure.spring.boot.version>
+    <azure.spring.boot.cosmos.version>3.6.0</azure.spring.boot.cosmos.version>
```

Comment:
[Curious] Why do you put the versions in the property and not directly in the dependency? Do you use any tool to generate this?

Reply:
What kind of validation is done on the BOM closure?

Reply:
> [Curious] Why do you put the versions in the property and not directly in the dependency? Do you use any tool to generate this?

No, I edit it manyally.

I do like this is to keep the same style to other items.

I guess this file do like this is to keep the same style with other poms, like [spring-boot-dependencies]( https://repo1.maven.org/maven2/org/springframework/boot/spring-boot-dependencies/2.5.2/spring-boot-dependencies-2.5.2.pom).

I think each method is OK: put the versions in the property or directly in the dependency.


@saragluna Do you know any other special reasons?

Reply:
> What kind of validation is done on the BOM closure?

We have [azure-boms-ci](https://dev.azure.com/azure-sdk/public/_build/results?buildId=993891&view=logs&j=09375a66-266a-5506-6757-13d166d199ef&t=51a6bf5b-7f00-542d-6b14-37ca4f37abfa) pipeline. But seems it can not check `dependency not found` error.